pom.xml

```diff
@@ -42,7 +42,7 @@
                             <!-- Generate a file header with timestamp -->
                             <noFileHeader>false</noFileHeader>
                             <!-- Delete generateDirectory before recreating -->
-                            <removeOldOutput>true</removeOldOutput>
+                            <removeOldOutput>false</removeOldOutput>
```

Comment:
I'm guessing this change was from testing?  I think we want this as `true`, otherwise old content could remain in the `generateDirectory` when you next regenerate the API.  If I'm misunderstanding, then we should change the comment here as we are no longer deleting the generatDirectory